dnf-behave-tests/features/ssl.feature

```diff
@@ -49,7 +51,7 @@ Scenario: Installing a package using untrusted client cert should fail
         # for stderr containing the expected output and ignore the superfluous
         # errors.
     And stderr contains "Errors during downloading metadata for repository 'dnf-ci-fedora':"
-    And stderr contains "  - Curl error \(56\): Failure when receiving data from the peer for https://localhost:[0-9]+/repodata/repomd.xml \[OpenSSL SSL_read: error:14094418:SSL routines:ssl3_read_bytes:tlsv1 alert unknown ca, errno 0\]"
+    And stderr contains "  - Curl error \(56\): Failure when receiving data from the peer for https://localhost:[0-9]+/repodata/repomd.xml"
```

Comment:
Are you sure this will help?

If I am reading the confusing console logs correctly the stderr contains only:
```
Errors during downloading metadata for repository 'dnf-ci-fedora':
  - Curl error (55): Failed sending data to the peer for https://localhost:36203/repodata/repomd.xml [SSL_write() returned SYSCALL, errno = 32]
  - Curl error (55): Failed sending data to the peer for https://localhost:36203/repodata/repomd.xml [SSL_write() returned SYSCALL, errno = 104]
Error: Failed to download metadata for repo 'dnf-ci-fedora': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
```
So it still won't match right?

Reply:
Ech, right. Let's leave out all the error details then...